Main_Project/src/main/java/com/italent/controller/ItalentDemo.java

```diff
@@ -0,0 +1,136 @@
+package com.italent.controller;
+
+import java.io.FileInputStream;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Properties;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.impl.client.BasicResponseHandler;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.italent.dao.LithiumMessageDAO;
+import com.italent.model.LithiumMessage;
+import com.italent.utils.LithiumUtils;
+
+public class ItalentDemo {
+    public static final Logger logger = LogManager.getLogger(ItalentDemo.class.getName());
+    public static void main(String[] args) throws Exception {
+        Properties prop = new Properties();
+        prop.load(new FileInputStream("./config/application.props"));
+        String accTokenUrl = prop.getProperty("GET_SESSION_KEY_URL");
+        logger.info("Lithium session key api : " + accTokenUrl);
+        LithiumUtils lithiumUtils = new LithiumUtils();
+        String accToken = generateSessionKey(accTokenUrl, lithiumUtils);
+        String checkBoard = checkingBoard(lithiumUtils, accToken);
+        if (null != checkBoard) {
+            createMessage(lithiumUtils, checkBoard, accToken);
+        } else {
+            String boardId = createBoard(lithiumUtils, accToken);
+            createMessage(lithiumUtils, boardId, accToken);
+        }
+    }
+
+    private static String checkingBoard(LithiumUtils lithiumUtils, String accToken) {
+        try {
+            Properties prop = new Properties();
```

Comment:
Dont create properties file object at every method, already commented below check with that